src/Containers/List/ListToggle.tsx

```diff
@@ -10,12 +10,14 @@ const LIST_TOGGLE_LEFT = AngleLeft;
 interface ListToggleProps extends ButtonProps {
     isOpen: boolean;
     setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    hideToggleTablet?: boolean;
```

Comment:
IsToggleTabletHidden

Always start with is for Boolean.

Reply:
IsHiddenTabletToggle etc

Reply:
Changed to isToggleHiddenDesktop